x-pack/osquerybeat/internal/ecs/mapping.go

```diff
@@ -1,3 +1,7 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
 package ecs
```

Comment:
@aleksmaus this was missing in https://github.com/elastic/beats/pull/24456 breaking the docs build, we are checking why `check-headers` was not triggered in the PR.

Reply:
Thank you for the fixing this!